engine/gamesys/src/gamesys/components/comp_model.cpp

```diff
@@ -139,9 +140,14 @@ namespace dmGameSystem
         dmGraphics::VertexElement ve[] =
         {
                 {"position", 0, 3, dmGraphics::TYPE_FLOAT, false},
-                {"texcoord0", 1, 2, dmGraphics::TYPE_FLOAT, false},
-                {"normal", 2, 3, dmGraphics::TYPE_FLOAT, false},
+                {"normal", 1, 3, dmGraphics::TYPE_FLOAT, false},
+                {"tangent", 2, 3, dmGraphics::TYPE_FLOAT, false},
+                {"color", 3, 4, dmGraphics::TYPE_FLOAT, false},
+                {"texcoord0", 4, 2, dmGraphics::TYPE_FLOAT, false},
+                {"texcoord1", 5, 2, dmGraphics::TYPE_FLOAT, false},
```

Comment:
In the future we might want specific declarations depending on the actual data coming in. Here _all_ models have the same attributes.